### ISSUE TITLE
"AS" is used in lesson before it is introduced

### ISSUE THREAD

**Issue by amyehodge** [datacarpentry/sql-ecology-lesson#190]
Using "AS" for reported field name aliases is included as part of the "HAVING" keyword section of 02-Aggregation, but it is not introduced and discussed until the 03-Joins and Aliases module in the lesson materials. As aliases seem like a lower priority topic, I would recommend moving how they interact with HAVING to the alias section and out of the HAVING section.

Issue #189 mentions a different type of alias issue.

**Comment by orchid00** [comment]
Very recently I updated some exercises that had AS before the concept was introduced. Would you like to suggest a PR?

**Comment by orchid00** [comment]
I've checked more in detail now. I see "AS" is being used as part of HAVING and also as part of CREATE VIEW viewname AS. The create view is one of the key concepts of this section 02-Aggregation. I will then, propose to move "Aliases" concept and examples from section  03 to 02. I also think that having JOINS as a separated lesson on its own would be good. This would need many parts moved around. Would anyone like to suggest something different? before starting such changes. What do you think @amyehodge

**Comment by orchid00** [comment]
Would you like to comment, regarding my last comment? @tracykteal @ErinBecker @tpoisot

**Comment by orchid00** [comment]
I will work on that, this is sitting here for too long. Thanks @amyehodge

**Comment by orchid00** [comment]
Hi Tim, @tpoisot I got the OK from @ErinBecker to move the Alias section from 03 to 02. I will do a fair bit of changes in these two episodes. It was also suggested to write a post to let the community know about this change. Would this post be something you would like to help with? It would be aimed to the community informing about the changes, and also to other maintainers letting them know the process. 

Question: to @ErinBecker the name on episode 03 is currently 03-sql-joins-aliases.md
is it possible to update this name to 03-sql-joins.md once all the changes are merged? If I change the name of the document what other links should we check?

**Comment by ErinBecker** [comment]
Thanks @orchid00 for the communication about this. Yes, please update the episode name (and file name) for accuracy. It would be useful to check the repo for any other occurrences of that link name and update those as well (there may be some instances of "in a previous lesson" or "in a later lesson" that reference this episode). I can't think of any external links (outside of the lesson) that would be affected, but I will keep an eye out for these.

**Comment by orchid00** [comment]
i renamed the file and did a check, it seems like the links are working fine. 
I will leave this open, for a few days, to see if something needs to be fixed.

**Comment by remram44** [comment]
Fixed by 756ac399.
